test/e2e/edu-helpers.js

```diff
@@ -15,6 +18,56 @@ function initApplicationSubmitMock(form) {
       }
     }
   });
+  /* eslint-disable camelcase */
+  mock(token, {
```

Comment:
What's all this for?

Reply:
When I ran the test locally, the buttons weren't showing up, which appeared to be because the login requests all failed.